hangupsbot/plugins/telesync/__init__.py

```diff
@@ -349,76 +367,93 @@ def tg_on_photo(tg_bot, tg_chat_id, msg):
     if str(tg_chat_id) in tg2ho_dict:
         ho_conv_id = tg2ho_dict[str(tg_chat_id)]
 
-        photos = tg_util_get_photo_list(msg)
-        photo_caption = tg_util_get_photo_caption(msg)
+        chat_title = tg_util_get_group_name(msg)
 
-        photo_id = photos[len(photos) - 1]['file_id']  # get photo id so we can download it
+        user = tg_util_sync_get_user_name(msg)
+        text = "uploading photo from <b>{}</b> in <b>{}</b>...".format(
+            user,
+            chat_title )
 
-        # TODO: find a better way to handling file paths
-        photo_path = 'hangupsbot/plugins/telesync/telesync_photos/' + photo_id + ".jpg"
+        yield from tg_bot.chatbridge._send_to_internal_chat(
+            ho_conv_id,
+            text,
+            {   "config": config,
```

Comment:
fixed by 7c13303 - regression introduced during chatbridge refactor, stickers/images will now be propagated across relays properly